web/thesauruses/lua/5/queues_stacks.json

```diff
@@ -0,0 +1,312 @@
+      "name": "Import statement to add circular queue",
+      "not-implemented": "true"
+    },
+    "circular_queue_data_type": {
+      "name": "Queue data type name",
+      "not-implemented": "true"
+    },
+    "circular_queue_create_statement": {
+      "name": "Create a queue",
+      "not-implemented": "true"
+    },
+    "circular_queue_create_copy_statement": {
+      "name": "Create a queue from existing queue",
+      "not-implemented": "true"
+    },
+    "circular_queue_create_copy_from_list_statement": {
+      "name": "Create a queue from a list of items",
+      "not-implemented": "true"
+    },
+    "circular_queue_destroy_statement": {
+      "name": "Destroy/delete a queue",
+      "not-implemented": "true"
+    },
+    "circular_queue_data_structure": {
+      "name": "Data structure that backs the simple queue",
+      "not-implemented": "true"
+    },
+    "priority_queue_import_statement": {
+      "name": "Import statement to add priority queue",
+      "not-implemented": "true"
+    },
+    "priority_queue_data_type": {
+      "name": "Queue data type name",
+      "not-implemented": "true"
+    },
+    "priority_queue_create_statement": {
+      "name": "Create a queue",
+      "not-implemented": "true"
+    },
+    "priority_queue_create_copy_statement": {
+      "name": "Create a queue from existing queue",
+      "not-implemented": "true"
+    },
+    "priority_queue_create_copy_from_list_statement": {
+      "name": "Create a queue from a list of items",
+      "not-implemented": "true"
+    },
+    "priority_queue_destroy_statement": {
+      "name": "Destroy/delete a queue",
+      "not-implemented": "true"
+    },
+    "priority_queue_data_structure": {
+      "name": "Data structure that backs the simple queue",
+      "not-implemented": "true"
+    },
+    "double_ended_queue_import_statement": {
+      "name": "Import statement to add double-ended queue",
+      "not-implemented": "true"
+    },
+    "double_ended_queue_data_type": {
+      "name": "Queue data type name",
+      "not-implemented": "true"
+    },
+    "double_ended_queue_create_statement": {
+      "name": "Create a queue",
+      "not-implemented": "true"
+    },
+    "double_ended_queue_create_copy_statement": {
+      "name": "Create a queue from existing queue",
+      "not-implemented": "true"
+    },
+    "double_ended_queue_create_copy_from_list_statement": {
+      "name": "Create a queue from a list",
+      "not-implemented": "true"
+    },
+    "double_ended_queue_destroy_statement": {
+      "name": "Destroy/delete a queue",
+      "not-implemented": "true"
+    },
+    "double_ended_queue_data_structure": {
+      "name": "Data structure that backs the simple queue",
+      "not-implemented": "true"
+    },
+    "stack_import_statement": {
+      "name": "Import statement to add stacks",
+      "not-implemented": "true"
+    },
+    "stack_data_type": {
+      "name": "Stack data type name",
+      "code": [

```

Comment:
This should probably be `not-implemented` and the code should be a comment instead.